docker-compose.yml

```diff
@@ -613,6 +615,8 @@ services:
       # 1 GB isn't enough for a single sanitizer build
       CCACHE_MAXSIZE: 2G
       Protobuf_SOURCE: "AUTO"
+      # GH-39973: Do not use debug memory pool for ASAN
+      ARROW_DEBUG_MEMORY_POOL: "none"
```

Comment:
Could you keep this list in alphabetical order?

Reply:
Thank you. Addressed.